DotNet/Submission/appsettings.json

```diff
@@ -13,10 +13,11 @@
     "DatabaseName": "linkReportDb"
   },
   "SubmissionServiceConfig": {
-    "ReportServiceUrl": "http://localhost:7110/api/Report/GetSubmissionBundle"
-  },
-  "FileSystemConfig": {
-    "FilePath": "/data/Submission"
+    "ReportServiceUrl": "http://localhost:7110/api/Report",
```

Comment:
Leave these as blank "" in the default settings, and put the "localhost" url values in the `appsettings.Development.json`